<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TileWidget</class>
 <widget class="QWidget" name="TileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="TilePage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Plots</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QPushButton" name="bSmithchart">
                  <property name="text">
                   <string>Smithchart</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="bXYplot">
                  <property name="text">
                   <string>XY-plot</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="bWaterfall">
                  <property name="text">
                   <string>Waterfall</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Split Tile</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QPushButton" name="bSplitV">
                  <property name="text">
                   <string>Vertical</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../icons.qrc">
                    <normaloff>:/icons/vertical.svg</normaloff>:/icons/vertical.svg</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="bSplitH">
                  <property name="text">
                   <string>Horizontal</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../icons.qrc">
                    <normaloff>:/icons/horizontal.svg</normaloff>:/icons/horizontal.svg</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="bClose">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Close Tile</string>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/close.svg</normaloff>:/icons/close.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ContentFullScreenMode"/>
     <widget class="QWidget" name="ContentPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bClose</sender>
   <signal>clicked()</signal>
   <receiver>TileWidget</receiver>
   <slot>closeTile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSplitH</sender>
   <signal>clicked()</signal>
   <receiver>TileWidget</receiver>
   <slot>splitHorizontally()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSplitV</sender>
   <signal>clicked()</signal>
   <receiver>TileWidget</receiver>
   <slot>splitVertically()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>splitVertically()</slot>
  <slot>splitHorizontally()</slot>
  <slot>closeTile()</slot>
 </slots>
</ui>
